tests/playwright-test/playwright-test-fixtures.ts

```diff
@@ -225,7 +225,7 @@ export const test = base.extend<Fixtures>({
       runResult = await runPlaywrightTest(baseDir, params, env);
       return runResult;
     });
-    if (testInfo.status !== testInfo.expectedStatus && runResult)
+    if (testInfo.status !== testInfo.expectedStatus && runResult && !process.env.PW_RUNNER_DEBUG)
```

Comment:
:*( I found this always useful that it printed the output.

Reply:
This prevents double test output when using `PW_RUNNER_DEBUG` env.